phpBB/includes/ucp/ucp_pm_viewfolder.php

```diff
@@ -602,3 +618,430 @@ function get_pm_from($folder_id, $folder, $user_id)
 		'rowset'	=> $rowset
 	);
 }
+
+/**
```

Comment:
I don't like this. Just leave the function inside `ucp_om_viewmessage.php` and `include` it where necessary. This change makes `ucp_pm_viewfolder.php` too big.

Reply:
Are you sure? I was thinking it would be easier this way if we used a form